app/src/main/res/values/strings.xml

```diff
@@ -249,11 +249,11 @@
     <string name="introduction_finished">All Set!</string>
     <string name="introduction_set_up_profile">Set Up Your Profile</string>
     <string name="count_milliseconds_short">%d ms</string>
-    <string name="use_recommended_fonts">Apply the Recommended Font</string>
+    <string name="use_recommended_fonts">Recommended Font</string>
     <string name="use_recommended_fonts_notice">Use the supplied custom font</string>
     <string name="waiting">Waiting…</string>
-    <string name="theme_optimize">AMOLED Optimization</string>
-    <string name="theme_optimize_summary">Use less power with darker colors</string>
+    <string name="theme_optimize">AMOLED</string>
```

Comment:
I still don't think this is better than `AMOLED Display Optimization`.

Reply:
I see it used elsewhere, and it makes sense over "Black".
My comment went missing, but the actual result of total white on pitch black bleeds light in the sensory domain (if the room is dark).
So it isn't optimized for the screen type, but it is a tradeoff for less battery use (because of the particular caracteristics of AMOLED screens). Then people aren't fooled into thinking total white on dark is "optimal".

Reply:
Alright, I understand the reasoning now. I can go with this for now.